samples/snippets/src/main/java/pubsub/CreatePushSubscriptionExample.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pubsub;
+
+// [START pubsub_create_push_subscription]
+
+import com.google.cloud.pubsub.v1.SubscriptionAdminClient;
+import com.google.pubsub.v1.ProjectSubscriptionName;
+import com.google.pubsub.v1.PushConfig;
+import com.google.pubsub.v1.Subscription;
+import com.google.pubsub.v1.TopicName;
+
+public class CreatePushSubscriptionExample {
+  public static void main(String... args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String subscriptionId = "your-subscription-id";
+    String topicId = "your-topic-id";
+    String pushEndpoint = "https://my-test-project.appspot.com/push";
+
+    createPushSubscriptionExample(projectId, subscriptionId, topicId, pushEndpoint);
+  }
+
+  public static void createPushSubscriptionExample(
+      String projectId, String subscriptionId, String topicId, String pushEndpoint)
+      throws Exception {
+    try (SubscriptionAdminClient subscriptionAdminClient = SubscriptionAdminClient.create()) {
+      TopicName topicName = TopicName.of(projectId, topicId);
+      ProjectSubscriptionName subscriptionName =
+          ProjectSubscriptionName.of(projectId, subscriptionId);
+      PushConfig pushConfig = PushConfig.newBuilder().setPushEndpoint(pushEndpoint).build();
```

Comment:
I haven't read the Java samples style guide, but the vertical builder syntax seems a bit more clear:
```
PushConfig pushConfig = PushConfig.newBuilder()
  .setPushEndpoint(pushEndpoint)
  .build();
```
?

Reply:
`mvn com.coveo:fmt-maven-plugin:format` runs style checks. I let it dictate format.